svg/attributes/data.json

```diff
@@ -0,0 +1,102 @@
+{
+  "svg": {
+    "attributes": {
+      "href": {
```

Comment:
You should remove that object as href data are already contributed in each individual element (it worth checking if the data are up to date there)